packages/ds/src/admin/form/InputText.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { HiOutlineExclamation } from "react-icons/hi";
+
+type OwnProps = {
+  disabled?: boolean;
+  active?: boolean;
+  error?: string;
+  name: string;
+  value?: string;
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+};
```

Comment:
Comme échangé w/ @C2Chandelier c'est plus clair pour les (autres) devs si on mets en premier les props requises puis les optionnelles. Qu'en penses-tu ?
```suggestion
type OwnProps = {
  name: string;
  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
  disabled?: boolean;
  active?: boolean;
  error?: string;
  value?: string;
};
```